script/scaffold/templates/config_flow_oauth2/integration/api.py

```diff
@@ -55,4 +55,4 @@ async def async_get_access_token(self):
         if not self._oauth_session.valid_token:
             await self._oauth_session.async_ensure_token_valid()
 
-        return self._oauth_session.token
+        return self._oauth_session.token["access_token"]
```

Comment:
Now the async and sync implementations are no longer in sync. 

Can you explain why you want to make this change?

Reply:
Ah I see now. Yeah the documentation probably also should have an update. 

I think in this case let's indeed update both sync + async to return the actual access token (like you did here) 👍